### ISSUE TITLE
PPA for Ubuntu 19.10 / eoan or update install script

### ISSUE THREAD

**Issue by jsmestad** [brave/brave-browser#6522]
## Description 
Now that Ubuntu 19.10 Eoan is released (10/17/19), the install script is no longer correct due to a missing PPA for the new release name.

I have been using the 19.04 Disco PPA successfully to this point, so an option would be to update the script to use `disco` for now till an `eaon` verison is ready.

## Steps to Reproduce

   1. Follow install guide, run `sudo apt install brave-browser{-beta,-nightly}`
   2. View error 
   3. 

## Actual result:
`E: Failed to fetch https://brave-browser-apt-beta.s3.brave.com/dists/eoan/InRelease  403  Forbidden [IP: 151.101.50.217 443]`

## Expected result:

A successful installation of the browser.

## Reproduces how often: 
100% of the time

## Brave version (brave://version info)
All repos: stable, beta, dev

**Comment by fmarier** [comment]
We're currently working (brave/devops#464) on deprecating these distro names entirely and replacing them with a single `stable` release name, which means that this will no longer need to be updated whenever a new distro version is released.

All of these distro names are actually pointing to the same exact packages today (i.e. there is a single `.deb` for all versions of Debian-based distros).

**Comment by kbsali** [comment]
Just for the record, I believe https://github.com/brave/brave-browser/releases/download/v0.70.118/brave-browser_0.70.118_amd64.deb is the latest stable .deb file for installing brave browser.

**Comment by srirambv** [comment]
@kbsali  That is on latest on Nightly channel. Release channel latest version is `0.69.135`. GitHub shows the most recent build (irrespective of channel) as the latest build.

**Comment by fmarier** [comment]
For anybody looking to install Brave on Ubuntu 19.10, follow these instructions:

```
sudo apt install apt-transport-https curl

curl -s https://brave-browser-apt-release.s3.brave.com/brave-core.asc | sudo apt-key --keyring /etc/apt/trusted.gpg.d/brave-browser-release.gpg add -

echo "deb [arch=amd64] https://brave-browser-apt-release.s3.brave.com/ bionic main" | sudo tee /etc/apt/sources.list.d/brave-browser-release-bionic.list

sudo apt update

sudo apt install brave-browser
```

The distro version ("bionic", "disco", "eaon") doesn't matter because they are all aliases of one another. There is a single Brave `.deb` package that works for all versions.

**Comment by fmarier** [comment]
New Ubuntu distro names missing from our repo script:
- `Focal`
- `Eoan`

**Comment by fmarier** [comment]
While the PR has been merged, the fix hasn't been deployed yet.

**Comment by kronikpillow** [comment]
I have this same issue on Pop OS 19.10, but the problem is not the distro release name, the problem is it's reporting that the repository is unsigned ... here is the actual error

```
sudo apt install apt-transport-https curl

curl -s https://brave-browser-apt-release.s3.brave.com/brave-core.asc | sudo apt-key --keyring /etc/apt/trusted.gpg.d/brave-browser-release.gpg add -

source /etc/os-release

echo "deb [arch=amd64] https://brave-browser-apt-release.s3.brave.com/ $UBUNTU_CODENAME main" | sudo tee /etc/apt/sources.list.d/brave-browser-release-${UBUNTU_CODENAME}.list

sudo apt update

sudo apt install brave-browser
```

I tried setting the repository to [trusted=yes] by editing /etc/apt/sources.list.d/brave-browser-release-eoan.list but then I get the following error

`E: Malformed entry 1 in list file /etc/apt/sources.list.d/brave-browser-release-eoan.list (URI parse)
E: The list of sources could not be read.`

**Comment by fmarier** [comment]
@kronikpillow Have you tried following the instructions in https://github.com/brave/brave-browser/issues/6522#issuecomment-545672208?

The instructions you followed in https://github.com/brave/brave-browser/issues/6522#issuecomment-551201329 won't work because it will try to grab the "eaon" distro name which doesn't exist yet. Using "bionic" instead of `$UBUNTU_CODENAME` will work and will get you the exact same `.deb`.

**Comment by kronikpillow** [comment]
I overlooked that comment, that worked, thank you :)

**Comment by fmarier** [comment]
While we're still waiting for the proper fix to be deployed (brave/brave-core#3750), we have added the [two new Ubuntu release names](https://github.com/brave/brave-browser/issues/6522#issuecomment-545949040) to the repo.
